source/core/types/ut_reporter_base.tps

```diff
@@ -60,7 +56,8 @@ create or replace type ut_reporter_base authid current_user as object(
   member procedure after_calling_suite(self in out nocopy ut_reporter_base, a_suite in ut_logical_suite),
 
   -- run hooks continued
-  member procedure after_calling_run (self in out nocopy ut_reporter_base, a_run in ut_run)
+  member procedure after_calling_run (self in out nocopy ut_reporter_base, a_run in ut_run),
+  not instantiable member procedure finalize(self in out nocopy ut_reporter_base)
```

Comment:
@pazus, or maybe it should have default (empty) implementation, like the rest of the methods?